plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -101,6 +93,15 @@
               When ASCII format is selected, Encryption Key may contain
               alphanumeric and special characters. Key must be 16 char long.
             type: str
+          message_authenticator_code_key:
+            description:
+            - Message key used to encrypt shared secret.
+            - Updation of message key is not possible.
+            - Required when encryption_scheme is provided.
+            - >
+              When ASCII format is selected, Message Authentication Code Key may contain
```

Comment:
As of now, we can remove "When ASCII format is selected". We can add this when we implement ASCII and Hexa